was/extension/WasCommandManager.ts

```diff
@@ -33,7 +33,7 @@ export class WasCommandManager {
                 this.nodecg.log.info(`Connected !was-manager to twitch channel "${channel}"`);
     
                 this.chatClient.getClient()?.onMessage((chan, _, message, _msg) => {
-                    if (chan === channel.toLowerCase() && message === WasCommandManager.COMMAND) {
+                    if (chan === channel.toLowerCase() && message.startsWith(`${WasCommandManager.COMMAND} `)) {
```

Comment:
```suggestion
                    if (chan === channel.toLowerCase() && message.startsWith(`${WasCommandManager.COMMAND}`)) {
```
The bot should also accept only the command. Not explicitly with a space behind. Right?

Reply:
True, but it shouldn't accept !wasser, so maybe split the text and check if the first element is the comment?

Reply:
Just use a regex with whitespace after that like: ^![wW]as\s

Reply:
```suggestion
                    if (chan === channel.toLowerCase() && message.match(new RegExp(`^${WasCommandManager.COMMAND}(\s|$)`))) {
```
Like that?

Reply:
Yeah - have a variable - save some things of the command in that varible and change it afterwards via regex. Would totaly recommend doing this because who needs to have understandable code.

Reply:
I thought about a better regex
```suggestion
                    if (chan === channel.toLowerCase() && message.startsWith(new RegExp(`^${WasCommandManager.COMMAND}\\s?.*`))) {
```

Reply:
I would highly recommend putting the Regexp in the varible - change the name to visualize that it is containing the regexp and then just use it with the regexp command.